resource/lookup_pipeline.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
         type: docker-image
         source:
           repository: quay.io/mojanalytics/lookup-tables-etl
-          tag: v0.0.10
+          tag: latest
```

Comment:
do we always want latest?

Reply:
Yeah, the image just copies data from github to Athena so don't want to be releasing this resource each time it is updated